docs/spec/bank/keepers.md

```diff
@@ -0,0 +1,130 @@
+## Keepers
+
+The bank module provides three different exported keeper interfaces which can be passed to other modules which need to read or update account balances. Modules should use the least-permissive interface which provides the functionality they require.
```

Comment:
Should the interfaces maybe go in the `sdk` (`types`) folder?  And the structs go in the `bank` module?